lib/multigrid.cpp

```diff
@@ -492,7 +499,7 @@ namespace quda
       param_coarse_solver->pipeline = 8;
 
       param_coarse_solver->maxiter = param.mg_global.coarse_solver_maxiter[param.level+1];
-      param_coarse_solver->Nkrylov = param_coarse_solver->maxiter < 20 ? param_coarse_solver->maxiter : 20;
+      param_coarse_solver->Nkrylov = param_coarse_solver->maxiter < param_coarse_solver->Nkrylov ? param_coarse_solver->maxiter : param_coarse_solver->Nkrylov;
```

Comment:
@maddyscientist I feel like this change caused controversy at some point---do you remember why? I think it was just because it changed some default behavior that was being assumed in Chroma, but I'm not sure.

Reply:
The "controversy" was over this line
```c++
param_coarse_solver->Nkrylov = param.mg_global.coarse_solver_ca_basis_size[param.level+1];
```
since it changed the prior behaviour when using the CA-GCR coarse solver.  I got over that though, and that's no problem now 😜